css/index.css

```diff
@@ -143,4 +211,211 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+.top-content-about {
+    border-bottom: 1px dashed black;
+    margin-bottom: 20px;
+    padding-bottom: 20px;
+}
+
+.about-content {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    width: 95%;
+    margin: 0 auto;
+}
+
+.about-content .about-points {
+    border-bottom: 1px dashed black;
+    width: 48%;
+    margin-bottom: 40px;
+    padding-bottom: 20px;
+}
+
+.about-content .about-points h2 {
+    margin-bottom: 10px;
+}
+
+.about-content .about-points button {
+    background: white;
+    vertical-align: top;
+    margin-top: 5px;
+    padding: 8px 25px;
+    font-size: 15px;
+    border: 2px solid black;
+    border-radius: 5px;
+    cursor: pointer;
+    margin-top: 15px;
+}
+
+.bottom-content-about {
+    margin-bottom: 20px;
+    padding-bottom: 20px;
+}
+
+.bottom-content-about .text-container {
+    margin: 0 auto;
+    width: 95%;
+}
+
+.products-content {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    width: 95%;
+    margin: 0 auto;
+}
+
+.products-content .products-items {
+    border-bottom: 1px dashed black;
+    width: 48%;
+    margin-bottom: 40px;
+    padding-bottom: 20px;
+}
+
+.products-content .products-items img {
+    width: 75%;
+}
+
+.products-content .products-items h2 {
+    margin-bottom: 10px;
+}
+
+.products-content .products-items button {
+    background: white;
+    vertical-align: top;
+    margin-top: 5px;
+    padding: 8px 25px;
+    font-size: 15px;
+    border: 2px solid black;
+    border-radius: 5px;
+    cursor: pointer;
+    margin-top: 15px;
+}
+
+.jumbo-prod {
+    margin-left: 10%;
+    margin-bottom: 30px;
+}
+
+.top-content-product {
+    border-bottom: 1px dashed black;
+    margin-bottom: 20px;
+    padding-bottom: 20px;
+}
+
+
+/* Animations */
+
+@keyframes footerbackground {
+    from {background-color: black;}
+    to {background-color: red;}
+}
+
+
+/* Contact Form */
+
+input[type=text], select, textarea {
+  width: 100%;
+  padding: 10px;
+  border: 1px solid blue;
+  border-radius: 5px;
+  box-sizing: border-box;
+  margin-top: 10px;
+  margin-bottom: 30px;
+  resize: vertical;
+}
+
+
+input[type=submit] {
+    background: blue;
+    color: white;
+    padding: 10px;
+    margin-bottom: 20px;
+}
+
+.thanks {
+    width: 95%;
+    margin: 0 auto;
+    margin-top: 80px;
+    margin-bottom: 30px;
+}
+
+.thanks h1 {
+    margin-bottom: 50px;
+}
+
+
+
```

Comment:
Nice job using @media queries especially since this is material that was not covered!!!